server/cmwell-ws/app/controllers/Application.scala

```diff
@@ -1534,13 +1534,13 @@ callback=< [URL] >
                 fSearchResult.flatMap { case (ok, searchResult) =>
                   extractFieldsMask(getQueryString("fields"),typesCache(nbg),cmwellRDFHelper, nbg).map { fieldsMask =>
                     // Prepare pagination info
-                    val linkBase = cmWellBase + normalizedPath + getQueryString("format").map {
-                      "?format=" + _
-                    }.getOrElse("?") + getQueryString("with-descendants").map {
+                    val linkBase = cmWellBase + normalizedPath + "?op=search" + getQueryString("format").map {
+                      "&format=" + _
+                    }.getOrElse("") + getQueryString("with-descendants").map {
```

Comment:
Use `Option.fold(ifEmpty)(elem => ...)` instead of `Option.map(elem => ...).getOrElse(ifEmpty)`.  
It performs a bit better and saves a few LOCs ;)